### DISCUSSION TITLE
我该如何像“回声洞”“你知道吗”主页预设那样，在自己的主页中添加可刷新的”回声洞“或”你知道吗“？

### DISCUSSION THREAD

**Discussion by MFn233** [Meloong-Git/PCL#2694] [提问] [answered]
正如标题所示，我创建了一个主页文件，想要在其中添加**”你知道吗“**等板块。

但是，我遇到了一个问题。我所创建的_Text_属性为_{hint}_的_TextBlock_**无法刷新**，我也并**不知道如何创建一个刷新按钮**

![image](https://github.com/Hex-Dragon/PCL2/assets/103323756/1ca217c5-d087-4340-9d0a-b24b6d1369d5)

> （这玩意是暂时摆在这里测试用的）

**Comment by MoYuan-CN** [comment, marked as answer]
创建一个主页刷新按钮：
``` xaml
<local:MyButton Text="刷新" EventType="刷新主页" />
```
当然，自己往上面加样式配置也是可以的

**Reply by MFn233** [reply]
感谢
